content/docs/conditional-rendering.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: conditional-rendering
-title: Conditional Rendering
+title: 條件渲染
```

Comment:
Hey @ivan-ha ! We decide to keep `rendering` in English because 渲染 also means [3D modeling](https://zh.wikipedia.org/wiki/%E6%B8%B2%E6%9F%93). Checkout our wiki.

Reply:
sure, will change to `條件 rendering` instead